<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommitDialog</class>
 <widget class="QDialog" name="CommitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commit</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="summaryRowLayout">
     <item>
      <widget class="QLineEdit" name="summaryEditor">
       <property name="placeholderText">
        <string>Enter commit summary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="counterLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="descriptionEditor">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Long-form description (optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="revealAuthor">
     <property name="text">
      <string>&amp;Edit author</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="authorGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="SignatureForm" name="authorSignature" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="detachedHeadWarning">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Warning: You are not in any branch (detached HEAD). You should create a branch to ensure your commit wonâ€™t be lost inadvertently.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignatureForm</class>
   <extends>QWidget</extends>
   <header>gitfourchette.widgets.signatureform</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>summaryEditor</tabstop>
  <tabstop>descriptionEditor</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CommitDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CommitDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>revealAuthor</sender>
   <signal>toggled(bool)</signal>
   <receiver>authorGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>revealAuthor</sender>
   <signal>toggled(bool)</signal>
   <receiver>authorGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
